frontend/package.json

```diff
@@ -52,12 +52,12 @@
     "lodash-es": "^4.17.15",
     "node-gyp": "^7.1.2",
     "node-sass": "^4.14.1",
-    "react": "^16.13.1",
+    "react": "^17.0.2",
```

Comment:
Wanted 18... PF is not ready.
https://github.com/patternfly/patternfly-react/issues/7142